### PR TITLE
build: pull sbt-paradox to 0.6.2

### PR DESCRIPTION
**Pull request by jrudolph**
That's to fix the nightly build against akka master which now also uses
paradox 0.6.1.

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -18,8 +18,9 @@ addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.9.0") // for advan
 addSbtPlugin("io.spray" % "sbt-boilerplate" % "0.6.1")
 addSbtPlugin("org.foundweekends" % "sbt-bintray" % "0.5.2")
 addSbtPlugin("com.lightbend.sbt" % "sbt-javaagent" % "0.1.4")
-addSbtPlugin("com.lightbend.akka" % "sbt-paradox-akka" % "0.18")
+addSbtPlugin("com.lightbend.paradox" % "sbt-paradox" % "0.6.1")
 addSbtPlugin("com.lightbend.paradox" % "sbt-paradox-apidoc" % "0.1+9-d846d815")
```
Comment (jrudolph) [inline project/plugins.sbt]: @raboof do we still need to use that snapshot version?

Comment (raboof) [reply project/plugins.sbt]: No, looks like we can depend on 0.2 now

File: project/plugins.sbt [hunk 2]
```diff
@@ -18,8 +18,9 @@ addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.9.0") // for advan
 addSbtPlugin("io.spray" % "sbt-boilerplate" % "0.6.1")
 addSbtPlugin("org.foundweekends" % "sbt-bintray" % "0.5.2")
 addSbtPlugin("com.lightbend.sbt" % "sbt-javaagent" % "0.1.4")
+addSbtPlugin("com.lightbend.paradox" % "sbt-paradox" % "0.6.1")
+addSbtPlugin("com.lightbend.paradox" % "sbt-paradox-apidoc" % "0.2")
 addSbtPlugin("com.lightbend.akka" % "sbt-paradox-akka" % "0.18")
```
Comment (helena) [inline project/plugins.sbt]: This one is up to 0.20 now but side note: https://github.com/akka/akka/pull/27473#issuecomment-519795226

**Comment by johanandren** [comment]
```
[info]   _: java.lang.IllegalStateException: Duplicate anchor [#improvements-in-docs] on [release-notes/10.1.x.html]
```

**Comment by raboof** [comment]
Fixed in the next release, https://github.com/lightbend/paradox/pull/341

**Comment by jrudolph** [comment]
Woops, seems we did the same work here

File: project/plugins.sbt (hunk 2 above)
Comment (jrudolph) [reply project/plugins.sbt]: Yep, we should do that in another step and visually check the change.

**Pull request merged**